pyproject.toml

```
@@ -30,9 +30,7 @@ dependencies = [
     "djangorestframework-dataclasses==1.3.*",
     "django-nonrelated-inlines==0.2.*",
     "django-filter==24.1",
-    # FIXME: Go back to PyPI when https://github.com/bhch/django-jsonform/pull/162 is available on PyPi
-    #  https://github.com/nycmeshnet/meshdb/issues/526
-    "django-jsonform@git+https://github.com/willnilges/django-jsonform.git@51cbed42ccdaec81a35c97a919d054e2c9ca0207",
+    "django-jsonform==2.23.0",
```

```suggestion
    "django-jsonform==2.23.*",
```

We nearly always want patch versions without manual action for security and bugfixes (`2.23.1` is actually available already so we definitely want this)